yarn-project/cli-wallet/src/utils/options/fees.ts

```diff
@@ -211,3 +213,20 @@ function parseGasLimits(gasLimits: string): { gasLimits: Gas; teardownGasLimits:
     teardownGasLimits: new Gas(parsed.teardownDA, parsed.teardownL2),
   };
 }
+
+function parseGasFees(gasFees: string): { maxFeesPerGas: GasFees } {
+  const parsed = gasFees.split(',').reduce((acc, fee) => {
+    const [dimension, value] = fee.split('=');
+    acc[dimension] = parseInt(value, 10);
+    return acc;
+  }, {} as Record<string, number>);
+
+  const expected = ['da', 'l2'];
```

Comment:
Are you sure we want to expect DA gas here?

Reply:
Probably not, were mainly for completeness.